Makefile

```diff
@@ -24,7 +24,7 @@ build:
 build_race:
 	CGO_ENABLED=0 go build -race $(BUILD_FLAGS) -tags '$(BUILD_TAGS)' -o build/tendermint ./cmd/tendermint
 
-install:
+install:aw
```

Comment:
what's aw?

Reply:
um... probably 'append' 'word' oh shit I'm in the wrong window and insert mode :)